cli/templates/tests/index.md

```diff
@@ -129,10 +129,12 @@ The file supports complex values and can even refer to an OAuth configuration:
 }
 ```
 
+## Sensitive Values
+
```

Comment:
Tady jen fix že ty palceholdery pro env proměnný v testech templates můžou být použitý všude, nejen v inputs.json.